search/routes.test.js

```diff
@@ -0,0 +1,63 @@
+import { describe, it, beforeEach, afterEach, mock } from 'node:test';
+import assert from 'node:assert';
+import request from 'supertest';
+
+import createDBQuery from '../db/index.js';
+
+describe('search-service router', () => {
+    let app
+
+    beforeEach(async () => {
+        const fakeResults = { hits: { hits: [{ _id: 1 }] } };
+        const searchSpy = mock.fn(async () => fakeResults);
+        class FakeClient {
+            constructor() {}
+            search = searchSpy;
+        }
+        const fakeLogger = fn(() => {});
```

Comment:
const fakeLogger = fn(() => {});
is fn imported? Or should it be mock.fn as above?